cmd/hugo-post-preview/hugo-post-preview.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
Is this the idiomatic way to build binaries and CLI tools on Go? In particular, what package should they be in and is this directory tree ok?

Reply:
I've followed the blog post at https://ieftimov.com/post/golang-package-multiple-binaries/ and renamed this file to `main.go` accordingly.

Reply:
cmd/<binary>/main.go

example
https://github.com/kubernetes/kubernetes/tree/master/cmd

Reply:
It's fine then, awesome! :smile: